components/Nav/Navbar/MobileSocialLogos.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Icon, { GithubOutlined, TwitterOutlined } from '@ant-design/icons';
+import styled from 'styled-components';
+//@ts-ignore
+import DiscordLogo from 'public/img/logos/socials/discord-logo-white.svg';
+//@ts-ignore
+import DiscourseLogo from 'public/img/logos/socials/discourse-logo-white.svg';
```

Comment:
Differently sourced images

Reply:
yeah its just how to work with antd/Icon so all the styles get applied, alternatively we could get all the socials icons https://ant.design/components/icon/